docs/user-manuals/multi-hierarchy-elastic-quota-management.md

```diff
@@ -616,6 +618,12 @@ rules:
   - patch
   - update
   - delete
```

Comment:
Not sure if the "delete" verb of pods should be removed or not, help me here please.

Reply:
`Delete` is used by default preemption.